From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Sofiane H. Djerbi" <46628754+kugge@users.noreply.github.com>
Date: Thu, 11 May 2023 04:52:48 +0300
Subject: [PATCH] Add SIMD utilities

Patch from Pufferfish

The utilities added in this patch are not intended to be used by plugins
and API spec stability is NOT guaranteed. If you use this in plugins,
they WILL break eventually.

diff --git a/src/main/java/gg/pufferfish/pufferfish/simd/SIMDChecker.java b/src/main/java/gg/pufferfish/pufferfish/simd/SIMDChecker.java
new file mode 100644
index 0000000000000000000000000000000000000000..f96c0dcc25ae90b58de861082aaad2a323e49579
--- /dev/null
+++ b/src/main/java/gg/pufferfish/pufferfish/simd/SIMDChecker.java
@@ -0,0 +1,2 @@
+package gg.pufferfish.pufferfish.simd;public class SIMDChecker {
+}
diff --git a/src/main/java/gg/pufferfish/pufferfish/simd/SIMDDetection.java b/src/main/java/gg/pufferfish/pufferfish/simd/SIMDDetection.java
new file mode 100644
index 0000000000000000000000000000000000000000..ac65490b8fb94b1933f53ab1433fd887e02bed47
--- /dev/null
+++ b/src/main/java/gg/pufferfish/pufferfish/simd/SIMDDetection.java
@@ -0,0 +1,2 @@
+package gg.pufferfish.pufferfish.simd;public class SIMDDetection {
+}
